.codecov.yml

```diff
@@ -7,6 +7,9 @@ coverage:
     project:
       default:
         threshold: 1%
+    patch:
+      default:
+        threshold: 1%
```

Comment:
I think 1% threshold doesn't make sense for a codecov/patch check. Ideally it should be set to 100% (i.e. all changed lines in a PR should be somehow tested).

Reply:
@veloman-yunkan Yes, I have put 90% to give a bit freedom to the developer.

Reply:
I had messed up here confusing **threshold** with **target**. **threshold** must be set to 0 for patch status check, and **target** must be set to the desired coverage value of the affected fragments of code.